applications/luci-app-basicstation/Makefile

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the Apache License Version 2.0.
+# See https://www.apache.org/licenses/LICENSE-2.0 for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-basicstation
+PKG_VERSION:=2.0.6
```

Comment:
Why such versioning? Couldn't it be 1.0.0 as this is the first initial version, which could gets into this repository?

Reply:
Removing versioning is fine, as there is automatic versioning in LuCI. (most packages do not have manual versions at all)